CONTRIBUTING.md

```diff
@@ -3,7 +3,38 @@
 We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
-## Formatting
+## Formatting and Preparation
```

Comment:
Actually, I change my mind: I think there is value in telling people how to build the course in the README and then use the space here to tell people how to install extra tools needed if you want to _contribute_ to the course:


```suggestion
## Formatting

We require files to be formatted consistently. To ensure this, you need to install a few tools.
```